superset-frontend/src/SqlLab/components/QueryTable.jsx

```diff
@@ -52,6 +52,12 @@ const openQuery = id => {
   const url = `/superset/sqllab?queryId=${id}`;
   window.open(url);
 };
+const StyledButton = styled(Button)`
+  position: static;
+`;
+const StyledSQLCard = styled(Card)`
+  position: static;
```

Comment:
same thing here.. you can create a staticPosition style and reuse it if it's just one style, rather than the styled component.